docs/static/ea-integration-tutorial.asciidoc

```diff
@@ -0,0 +1,183 @@
+[[ea-integrations-tutorial]]
+=== Tutorial: {ls} `elastic_integration filter` to extend Elastic {integrations}
+++++
+<titleabbrev>Tutorial: {ls} `elastic_integration filter`</titleabbrev>
+++++
+
+You can use {ls} to transform events collected by {agent} using our {integrations-docs}[Elastic integrations]. 
+You get the benefits of Elastic integrations--such as the simplicity of ingesting data from a wide variety of data 
+sources and ensuring compliance with the {ecs-ref}/index.html[Elastic Common Schema (ECS)]--combined with the extra 
+processing power of {ls}. 
+
+This new functionality is made possible by the <<plugins-filters-elastic_integration,elastic_integration filter>> plugin.
+When you include the `elastic_integration` filter in your configuration, {ls} reads certain field values generated by the {agent}, 
+and uses them to apply the transformations from Elastic integrations. This allows you to to further process events in the Logstash pipeline before sending them to their 
+configured destinations.
+
+This tutorial walks you through adding the {integrations-docs}/crowdstrike-intro[Crowdstrike integration], using {ls} to 
```

Comment:
@flexitrev, would you like to highlight a different integration for this tutorial and use case?